data/sql/updates/pending_db_auth/rev_1629280349876853000.sql

```diff
@@ -0,0 +1,13 @@
+INSERT INTO `version_db_auth` (`sql_rev`) VALUES ('1629280349876853000');
+
+ALTER TABLE autobroadcast CHANGE COLUMN `text` `text_enUS` LONGTEXT NOT NULL;
```

Comment:
I'd prefer to keep this as "text" or "text_default" as imo it should default to this if client locale isn't found or something.

Mainly thinking of acore_string that has "content_defualt"

Reply:
The client locale will always be found. The only usability of a text_default that I can think of would be if the person is too lazy to fill the rest of the text locale columns.

Reply:
```suggestion
ALTER TABLE `autobroadcast` CHANGE COLUMN `text` `text_enUS` LONGTEXT NOT NULL;
```